theatre/studio/src/utils/tightJsonStringify.ts

```diff
@@ -0,0 +1,9 @@
+export function tightJsonStringify(
+  obj: any,
+  replacer?: ((this: any, key: string, value: any) => any) | undefined,
+) {
+  return JSON.stringify(obj, replacer, 2)
+    .replace(/^([\{\[])\n (\s+)/, '$1$2')
+    .replace(/(\n[ ]+[\{\[])\n\s+/g, '$1 ')
+    .replace(/\n\s*([\]\}])/g, ' $1')
+}
```

Comment:
This needs docs / examples or named regexs. Why would this be used instead of JSON.stringify? What does `tight` mean? No whitespace?